### ISSUE TITLE
Can't Merge Different FanFic Versions - Calibre Plugin

### ISSUE THREAD

**Issue by hchasens** [JimmXinu/FanFicFare#981]
Many authors on RR and other platforms sign exclusive contracts (e.g. w/ Kindle Unlimited) that require them to take down previously posted chapters. 

To the readers / FFF this often looks like the following:
- Pre-Contract: Chapters 1 - 150 w/ 150 total chapters
- Post-Contract: Chapters 100-165 w/ 65 total chapters

When you go to update these stories post-contract FFF warns you that there are fewer chapters and updating will rewrite your current book. This is an issue as you can't update content without effectively deleting large swaths of the book. Ideally, FFF should recognise that some chapters have been taken, others have been added, and append the new chapters without deleting the previous ones. At the least, it should be saved as another edition rather than forcing us to not update or delete old content.

**Comment by JimmXinu** [comment]
Raised and discussed in prior issues #968 #950 #949 #942.

TL;DR:
> FFF's fundamental paradigm doesn't have a way to keep chapters that aren't in the TOC.
>
> A nontechnical issue is the FanFicFare is meant for fanfiction. If an author has removed their content to make money on it, I'm not interested in helping work around that.

**Comment by hchasens** [comment]
This is understandable. I'm a bit of a data hoarder so it slipped my mind but it's important to protect small creators. Thank you for the explanation.
